client/ayon_houdini/startup/otls/AyonPub.1.1.hda/Driver_1AyonPub_8_81.1/CreateScript

```diff
@@ -0,0 +1,15 @@
+# Automatically generated script
+\set noalias = 1
+#
+#  Creation script for AyonPub::1.1 operator
```

Comment:
Can we update the full HDA to be just `AYONPublish` or `ayon_publish` so that it matches the naming conventions of the other HDAs in the `otls/` folder.

Reply:
updated it to ayon_publish to be like the others

Reply:
done